source/bdDetect.py

```diff
@@ -228,42 +261,27 @@ def _get_usbDevices(self) -> typing.List[typing.Dict]:
 	def _get_hidDevices(self) -> typing.List[typing.Dict]:
 		return list(hwPortUtils.listHidDevices(onlyAvailable=True))
 
-#: The single instance of the device info fetcher.
-#: @type: L{_DeviceInfoFetcher}
-deviceInfoFetcher = _DeviceInfoFetcher()
+
+deviceInfoFetcher: _DeviceInfoFetcher

```

Comment:
This pattern is more typical for initialized module level variables

```suggestion
deviceInfoFetcher: Optional[_DeviceInfoFetcher] = None
```